app/code/Magento/Review/Test/Unit/Model/ReviewTest.php

```diff
@@ -135,6 +135,9 @@ public function testAggregate()
         $this->assertSame($this->review, $this->review->aggregate());
     }
 
+    /**
+     * @deprecated
```

Comment:
WAT? :)

Reply:
This test now is not valid, because I have changed logic and added new tests for new logic

Reply:
@Den4ik I'm asking why it's not removed then, never saw a deprecated test before :)